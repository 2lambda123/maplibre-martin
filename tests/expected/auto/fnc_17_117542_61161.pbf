é
public.function_zxy_query	"	ú: &	"	¬"º5	"	∏8Ú3	"	¿;?	"	Æ1ﬁ)	"	Ë0à/	"	¿.ê=	"	Ç$Ù0	"	ñ%Ñ&	"	Ú;ö (Ä x