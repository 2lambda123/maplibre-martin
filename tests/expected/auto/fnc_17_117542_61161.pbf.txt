=============================================================
layer: 0
  name: public.function_zxy_query
  version: 2
  extent: 4096
  feature: 0
    id: (none)
    geomtype: point
    geometry:
      POINT(3726,2469)
    properties:
  feature: 1
    id: (none)
    geomtype: point
    geometry:
      POINT(2209,3422)
    properties:
  feature: 2
    id: (none)
    geomtype: point
    geometry:
      POINT(3612,3321)
    properties:
  feature: 3
    id: (none)
    geomtype: point
    geometry:
      POINT(3808,4088)
    properties:
  feature: 4
    id: (none)
    geomtype: point
    geometry:
      POINT(3159,2671)
    properties:
  feature: 5
    id: (none)
    geomtype: point
    geometry:
      POINT(3124,3012)
    properties:
  feature: 6
    id: (none)
    geomtype: point
    geometry:
      POINT(2976,3912)
    properties:
  feature: 7
    id: (none)
    geomtype: point
    geometry:
      POINT(2305,3130)
    properties:
  feature: 8
    id: (none)
    geomtype: point
    geometry:
      POINT(2379,2434)
    properties:
  feature: 9
    id: (none)
    geomtype: point
    geometry:
      POINT(3833,2061)
    properties:
